spec/unit/recipes/server_spec.rb

```diff
@@ -57,8 +57,8 @@
       it do
         expect(chef_run).to run_bash('update Chef trusted certificates store')
           .with(
-            code: "  cat /opt/pebble/test/certs/pebble.minica.pem >> /opt/chef/embedded/ssl/certs/cacert.pem\n  touch /opt/chef/embedded/ssl/certs/PEBBLE-MINICA-IS-INSTALLED\n",
-            creates: '/opt/chef/embedded/ssl/certs/PEBBLE-MINICA-IS-INSTALLED'
+            code: 'cat /opt/pebble/test/certs/pebble.minica.pem >> /opt/cinc/embedded/ssl/certs/cacert.pem; touch /opt/cinc/embedded/ssl/certs/PEBBLE-MINICA-IS-INSTALLED',
+            creates: '/opt/cinc/embedded/ssl/certs/PEBBLE-MINICA-IS-INSTALLED'
```

Comment:
Can you please update this test so that it works with either Chef or Cinc? You can look [here](https://github.com/osuosl-cookbooks/base/blob/master/spec/unit/recipes/chef_client_spec.rb#L29-L37) how we did it in the base cookbook.